src/Providers/DependencyInjectionProvider.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Joselfonseca\LighthouseGraphQLPassport\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Joselfonseca\LighthouseGraphQLPassport\Contracts\AuthModelFactory as AuthModelFactoryContract;
+use Joselfonseca\LighthouseGraphQLPassport\Factories\AuthModelFactory;
+
+class DependencyInjectionProvider extends ServiceProvider
```

Comment:
Can this be in the same service provider that exists already? what is the reason for it to be a separated service provider?

Reply:
Yes, it can.

Sometimes the DI instructions clutter up the service provider. So I started to put them in a dedicated service provider.

But I'll can merge it to the existing provider.